instance/models/mixins/openedx_storage.py

```diff
@@ -61,13 +68,29 @@ def _get_s3_settings(self):
             # using shared s3 buckets
             "EDXAPP_AWS_LOCATION": self.swift_container_name,
             "EDXAPP_DEFAULT_FILE_STORAGE": 'storages.backends.s3boto.S3BotoStorage',
+
+            # Set up S3 image backend
+            # https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L688
+            "EDXAPP_PROFILE_IMAGE_BACKEND": {
+                "class": "storages.backends.s3boto.S3BotoStorage",
+                "options": {
+                    "bucket": self.s3_bucket_name,
+                    "location": '{}/{}'.format(self.swift_container_name, 'profile-images'),
+                    "custom_domain": self.s3_custom_domain,
+                    "access_key": self.s3_access_key,
+                    "secret_key": self.s3_secret_access_key,
```

Comment:
Hmm. Since django-storages [defaults to the global ones](https://github.com/jschneier/django-storages/blob/1.4.1/storages/backends/s3boto.py#L206) can we get away with not passing the ones which are already set? I think only `location` and `headers` are unique here.

Reply:
You're right! Thanks for the tip :)